test/templates/notification/index.njk

```diff
@@ -0,0 +1,443 @@
+{% extends "layout.njk" %}
+{% from "./../format.njk" import format %}
+{% from "./../format.njk" import format %}
+
+{% block content %}
+
+<h1>Notification</h1>
+{% set example %}
+<div class="fd-notification">
+    <div class="fd-notification__header">
+        <div class="fd-notification__title">Default Notification Title</div>
+        <button class="btn-playground fd-notification__close"></button>
+    </div>
+    <div class="fd-notification__body">
+        <div class="fd-notification__content">
+            <div class="fd-notification__text">
+                <div class="fd-notification__description">
+                    Default Notification Description <br>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                </div>
+                <div class="fd-notification__metadata">
+                    Default Notification Metadata <br>
+                    Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                </div>
+            </div>
+        </div>
+        <div class="fd-notification__footer">
+            <button class="btn-playground">More Info</button>
+            <div class="fd-notification__actions">
+                <button class="btn-playground">Approve</button>
+                <button class="btn-playground">Reject</button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<br>
+<br>
+
+<div class="fd-notification fd-notification--success">
+    <div class="fd-notification__header">
+        <div class="fd-notification__indicator--success"></div>
+        <div class="fd-notification__title">This is the notifications title Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</div>
+        <button class="btn-playground fd-notification__close"></button>
+    </div>
+    <div class="fd-notification__body">
+        <div class="fd-notification__content">
+            <div class="fd-notification__avatar">
+                <span>WW</span>
```

Comment:
If you put a placeholder class on this -- like one of the ones on `layout.css`, it won't throw you off on the unnormalize checkbox.

Reply:
done